source/ProtonPack/ProtonPack.ino

```diff
@@ -4663,25 +4683,90 @@ void readEEPROM() {
 
     // Update the LED counts for the Proton Pack.
     updateProtonPackLEDCounts();
+
+    // Read our configration object from the EEPROM.
+    objConfigEEPROM obj_config_eeprom;
+    unsigned int i_eepromConfigAddress = EEPROM.length() / 2;
+
+    EEPROM.get(i_eepromConfigAddress, obj_config_eeprom);
+
+    if(obj_config_eeprom.stream_effects > 0 && obj_config_eeprom.stream_effects != 255) {
+      if(obj_config_eeprom.stream_effects > 1) {
+        b_stream_effects = true;
+      }
+      else {
+        b_stream_effects = false;
+      }
+    }
+
+    if(obj_config_eeprom.stream_effects > 0 && obj_config_eeprom.stream_effects != 255) {
+      if(obj_config_eeprom.stream_effects > 1) {
+        b_cyclotron_single_led = true;
+      }
+      else {
+        b_cyclotron_single_led = false;
+      }
+    }
+  }
+}
+
+void clearConfigEEPROM() {
+  // Clear out the EEPROM data for the configuration settings only.
+  unsigned int i_eepromConfigAddress = EEPROM.length() / 2;
```

Comment:
I kept thinking back to this all of last night. As you had pointed out, the size of the EEPROM is perfectly large enough to fit the data to be stored plus extra for future use, but the calculation feels arbitrary. My main concern is how do you choose a location should a 3rd EEPROM space be needed? Would that use a different calculation? If its decided that these locations must change (or do change somehow) it obviously throws off the stored values. The more long-term approach that feels right is picking a distinct base-2 value (32, 64, 128, etc) and using that as the marker for this secondary block of values.